src/App.js

```diff
@@ -1,6 +1,21 @@
-import React from 'react'
-import Reviews from './components/Reviews'
+import React, { useState } from 'react'
+import { BrowserRouter as Switch, Route } from 'react-router-dom'
```

Comment:
Pretty sure this was an autoimport, because this import doesn't look familiar at all to me. I will just import the switch here since I have the BrowserRouter in my index.js.